README.md

```diff
@@ -0,0 +1,198 @@
+Classy Test
+===========
+
+Meteor package which provides a class-based wrapper around `tinytest`.
+
+It has the following features:
+
+ * Class-based test cases.
+ * Common setUp/tearDown methods for test cases, separated for server and client side.
+ * A single test can interleave client-side and server-side assertions.
+ * Support for asynchronous tests.
+ * Compatible with tinytest (all tests are actually registered via tinytest).
+
+Installation
+------------
+
+```
+meteor add peerlibrary:classy-test
+```
+
+Test cases
+----------
+
+Each test case is a class extending from `ClassyTestCase` as follows:
+
+```coffeescript
+class SimpleTestCase extends ClassyTestCase
+  testThatTrueIsTrue: =>
+    @assertTrue true, "True should be true."
+
+  testThatFalseIsFalse: =>
+    @assertFalse false, "False should be false."
+
+# Register the test case.
+ClassyTestCase.addTest new SimpleTestCase()
+```
+
+This simple test case definition will generate two tests via tinytest, the first will be called `SimpleTestCase - ThatTrueIsTrue` and the second one `SimpleTestCase - ThatFalseIsFalse`.
```

Comment:
Hm, not sure how you get class names and function names, but you know that when code is minimized, that could get distorted? Not sure if this is the case in tests as well, but at least I had problems with this in PeerDB (this is why you have to manually declare names). Also, `.name` property on functions and class constructors exist only on V8 based environments (Chrome and node.js). Try running tests in Firefox. Or do you register all names from the server? I do not think so.